src/main/java/mate/controller/DeleteManufacturerController.java

```diff
@@ -0,0 +1,24 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.service.ManufacturerService;
+
+public class DeleteManufacturerController extends HttpServlet {
+    private static final String MAIN_PAGE = "/index";
+    private static final Injector injector = Injector.getInstance("mate");
+    private final ManufacturerService manufacturerService
+            = (ManufacturerService) injector.getInstance(ManufacturerService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        Long id = Long.valueOf(req.getParameter("id"));
```

Comment:
Consider adding validation for the `id` parameter to avoid NumberFormatException. 🚨